kits/flutter_ume_kit_dio/lib/src/widgets/custom_message_page.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_ume_kit_dio/src/net/dio_manager.dart';
+
+// 自定义信息输入页面
```

Comment:
Please write code comments in English. And this comment should start with `///`. It indicate that it's a documentation comment.